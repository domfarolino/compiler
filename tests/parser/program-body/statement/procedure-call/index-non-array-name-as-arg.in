program dominic is

  integer a;
  string b;

  procedure fn(integer arg1 in, string arg2 in)
  begin
  end procedure;

begin
  fn(a[10], b);
end program.
