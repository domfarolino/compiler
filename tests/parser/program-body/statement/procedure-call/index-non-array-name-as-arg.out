[1;31mLine 11: Cannot index into non-array name 'a'
Line 11: Error parsing program body
[0m